spring-integration-hazelcast/src/main/java/org/springframework/integration/hazelcast/leader/LeaderInitiator.java

```diff
@@ -263,30 +273,37 @@ protected class LeaderSelector implements Callable<Void> {
 
 		protected final String role = LeaderInitiator.this.candidate.getRole();
 
-		private volatile boolean locked = false;
+		private volatile boolean leader = false;
 
 		@Override
 		public Void call() {
 			try {
 				while (isRunning()) {
 					try {
-						if (LeaderInitiator.this.lock.isLocked()) {
+						if (logger.isTraceEnabled()) {
+							logger.trace("Am I the leader (" + LeaderInitiator.this.candidate.getRole() + ") ? "
+									+ this.leader);
+						}
+						if (LeaderInitiator.this.getLock().isLockedByCurrentThread()) {
```

Comment:
We don't use prefixed for method calls, unless they are `static`.

Reply:
Hmm I am not sure If I follow?

Reply:
It has to be just `getLock().isLockedByCurrentThread()` because this method is not `static` and we use `this.` only for properties.